tests/tools/oecert/host/host.cpp

```diff
@@ -418,6 +430,26 @@ static oe_result_t _read_key(const char* filename, uint8_t** data, size_t* size)
 static oe_result_t _process_params(oe_enclave_t* enclave)
 {
     oe_result_t result = OE_FAILURE;
+    bool sgx_asem_env_set = false;
```

Comment:
We should restore whatever is set/unset to SGX_AESM_ADDR. I.e.

char* sgx_aesm_env = getenv(SGX_AESM_ADDR);
...
setenv(SGX_AESM_ADDR, agx_aesm_env, 1);  // Restore value

Reply:
Updated.